Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/server/DataDonationServer.kt

```diff
@@ -0,0 +1,35 @@
+package de.rki.coronawarnapp.datadonation.survey.server
+
+import com.google.protobuf.ByteString
+import dagger.Lazy
+import de.rki.coronawarnapp.datadonation.OneTimePassword
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.EdusOtp
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import timber.log.Timber
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class DataDonationServer @Inject constructor(
+    private val dataDonationApi: Lazy<DataDonationApiV1>
+) {
+
+    private val api: DataDonationApiV1
+        get() = dataDonationApi.get()
+
+    suspend fun authOTP(
+        data: OneTimePassword
+    ) = withContext(Dispatchers.IO) {
```

Comment:
Can you specify the return value?
It's `DataDonationResponse` right? but from reading the code without an IDE it's not obvious.

Reply:
caring for non-ide readers